tests/_files/Report/HTML/PathCoverageForBankAccount/BankAccount.php_path.html

```diff
@@ -386,23 +387,6 @@ <h5 class="structure-heading"><a name="BankAccount-&gt;withdrawMoney">BankAccoun
 
 </tbody>
 </table>
-
-
-   <footer>
-    <hr/>
-    <h4>Legend</h4>
-    <p><span class="success"><strong>Fully covered</strong></span><span class="warning"><strong>Partially covered</strong></span><span class="danger"><strong>Not covered</strong></span></p>
-    <p>
-     <small>Generated by <a href="https://github.com/sebastianbergmann/php-code-coverage" target="_top">php-code-coverage %s</a> using <a href="%s" target="_top">%s</a> at %s.</small>
-    </p>
-    <a title="Back to the top" id="toplink" href="#">
-        <svg xmlns="http://www.w3.org/2000/svg" width="12" height="16" viewBox="0 0 12 16"><path fill-rule="evenodd" d="M12 11L6 5l-6 6h12z"/></svg>
-    </a>
-   </footer>
-  </div>
-  <script src="_js/jquery.min.js" type="text/javascript"></script>
-  <script src="_js/popper.min.js" type="text/javascript"></script>
-  <script src="_js/bootstrap.min.js" type="text/javascript"></script>
-  <script src="_js/file.js" type="text/javascript"></script>
+%a
```

Comment:
This is needed in order to avoid the `preg_match` error. I think this footer part doesn't add value to the overall test suite, so it should be safe

Reply:
I agree that the footer does not add value. Thanks!